tests/type-tests/test/orchestration.test-d.ts

```diff
@@ -39,12 +36,12 @@ expectType<CompletionPostResponse>(
   (
     await new OrchestrationClient({
       templating: {
-        template: [{ role: 'user', content: 'Hello!' }]
+        defaults: { name: 'Bob' }, 
       },
       llm: {
         model_name: 'gpt-4o'
       }
-    }).chatCompletion()
+    }).chatCompletion({ messages: [{ role: 'user', content: 'Hello! {{?name}}' }]})
```

Comment:
I agree that we should allow variables also in `messages` in favor of grounding.